ethcore/src/client/evm_test_client.rs

```diff
@@ -70,7 +70,8 @@ lazy_static! {
 	pub static ref HOMESTEAD: spec::Spec = ethereum::new_homestead_test();
 	pub static ref EIP150: spec::Spec = ethereum::new_eip150_test();
 	pub static ref EIP161: spec::Spec = ethereum::new_eip161_test();
-	pub static ref _METROPOLIS: spec::Spec = ethereum::new_metropolis_test();
+	pub static ref BYZANTIUM: spec::Spec = ethereum::new_byzantium_test();
```

Comment:
Why `CONSTANTINOPLE` is not here?

Reply:
There's no Constantinople spec yet. It will be added in a later PR